contracts/src/v0.8/dev/automation/2_1/KeeperRegistryBase2_1.sol

```diff
@@ -109,9 +109,11 @@ abstract contract KeeperRegistryBase2_1 is ConfirmedOwner, ExecutionPrevention {
   uint256 internal s_fallbackGasPrice;
   uint256 internal s_fallbackLinkPrice;
   uint256 internal s_expectedLinkBalance; // Used in case of erroneous LINK transfers to contract
+  address internal s_upkeepManager;
   mapping(address => MigrationPermission) internal s_peerRegistryMigrationPermission; // Permissions for migration to and fro
   mapping(uint256 => bytes) internal s_upkeepTriggerConfig; // upkeep triggers
-  mapping(uint256 => bytes) internal s_upkeepOffchainConfig; // general configuration preferences
+  mapping(uint256 => bytes) internal s_upkeepOffchainConfig; // general config set by users for each upkeep
+  mapping(uint256 => bytes) internal s_upkeepAdminOffchainConfig; // general config set by an administrative role
```

Comment:
'admin' config is confusing here since there's already a concept of upkeep admin which is the end user